packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -313,6 +310,57 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
   }
 }
 
+/// An picker default magnifier, grey rrect first introduced in iOS 14
```

Comment:
A default selection overlay for [CupertinoPicker]s. 

It draws a gray rounded rectangle to match the picker visuals introduced in iOS 14

Reply:
Thank you, it's updated now.